client/app/home/home.html

```diff
@@ -1,8 +1,11 @@
 <center>
+    <div class="row spacer" style="margin-top:50px"></div>
     <div>
-        <img id="spinit" src="assets/images/headerlogofront.png" width="300"/>
+        <!-- <img id="spinit" src="assets/images/headerlogofront.png" width="300"/> -->
+        <p style="font-family: Kanit; font-size:200px"><i>OPENEXP</i></p>
```

Comment:
general question, do we have a consensus on how we want to stylize `OpenEXP`. Consistently all caps, or mixed?